crates/bevy_text/src/text.rs

```diff
@@ -124,8 +124,16 @@ pub struct TextSection {
 impl TextSection {
     /// Create a new [`TextSection`].
     pub fn new(value: impl Into<String>, style: TextStyle) -> Self {
+        let text_value = value.into();
+        if style.font_size <= 0.0 {
```

Comment:
```suggestion
        if style.font_size <= 0.0 || style.font_size.is_infinite() || style.font_size.is_nan() {
```
Is this ok for a finite and nan check?

Reply:
Or the check can be reversed, by looking for size > 0.0 and using just .is_finite()

Reply:
Just use `<= 0 || !is_finite()`. That covers NaN and the infinities.